tailwind.config.js

```diff
@@ -173,7 +482,63 @@ module.exports = {
         },
       };
 
-      addUtilities(utilities, ["responsive"]);
+      addUtilities(utilities, ["lib", "DEFAULT", "responsive"]);
+    }),
+
+    /* =========================================================================
+      Variants
+      ========================================================================== */
```

Comment:
Can we just use 

```js
/* Variants */
```

Those banner comments takes up too much attention.

Reply:
Okay !